src/Http/src/Header/AcceptHeader.php

```diff
@@ -128,6 +112,7 @@ private function addItem($item): void
             $item = AcceptHeaderItem::fromString((string)$item);
         }
 
+        /** @var AcceptHeaderItem|array $item  */
```

Comment:
added union `/** @var AcceptHeaderItem|array $item  */` to make `if (!$item instanceof AcceptHeaderItem) {` next valid.